### PR TITLE
Merge current state of UI insto main

### PR DESCRIPTION
**Pull request by MaximumMaxxx**
This is the beginning of as somewhat functional UI. We now having moving between rooms, a basic wumpus implementation and a whole lot more.

This PR also includes the console UI project, although seeing as Silas has left that might be deprecated in the future.

### CODE REVIEW

**Review by 1118422ILHS — CHANGES_REQUESTED** [review]
I cannot allow the duplication of files. It needs to be easy to make changes to a file and this is stupid also

**Review by 1118422ILHS — CHANGES_REQUESTED** [review]
misspelling

**Review by ilhs1096069 — APPROVED** [review]
This will help :)

File: Core/WumpusCore/Controller/Controller.cs [hunk 1]
```diff
@@ -11,41 +13,50 @@ namespace WumpusCore.Controller
...
+        private Trivia.Trivia trivia;
 
-        private Controller()
+        /// <summary>
+        /// Instantiates a controller and setup the required stuff for global controller
+        /// </summary>
+        /// <param name="trviaFile">The path to the file you want to load trivia from. See Triva/Questions.json for format</param>
+        /// <param name="topologyDirectory">The directory to load map files from</param>
+        /// <param name="mapId">The mapid to load from the topologyDirectory. Format is map{n}.wmp where n is the mapId</param>
+        public Controller(string trviaFile, string topologyDirectory, ushort mapId)
```
[the remark this answers is not in the document]
Comment (havecookies) [reply Core/WumpusCore/Controller/Controller.cs]: fixed

**Pull request merged**